shared/app/tab-bar/index.render.desktop.js

```diff
@@ -41,6 +41,7 @@ const TabBar = ({onTabClick, selectedTab, username, badgeNumbers}: Props) => (
       <TabBarButton
         className="keybase-nav"
         badgeNumber={badgeNumbers[tab]}
+        isNav={true}
```

Comment:
this stuff is temporary. nav should just be its own thing and not use this component. this is so the folders tab isn't messed up